test/nvd/task/check_test.clj

```diff
@@ -44,10 +44,10 @@
   (update-db/-main "test/resources/self-test.json")
   (let [project (check/-main "test/resources/self-test.json")]
     (is (== 11.0 (get-in project [:nvd :fail-threshold])))
-    (is (== 5.0 (get-in project [:nvd :highest-score])))
+    (is (== 0 (get-in project [:nvd :highest-score])))
```

Comment:
slightly puzzled why this dropped from 7.5 to 5.0 and now to zero .. happy to take this for now, but think i will create an issue to update `self-test.json` so we have something that is > 0